aea/cli/reset_password.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Implementation of the 'aea reset_password' subcommand."""
+
+import click
+
+from aea.cli.registry.login import registry_reset_password
+
+
+@click.command(name="reset_password", help="Reset password of Registry account.")
+@click.argument("email", type=str, required=True)
+def reset_password(email):
```

Comment:
why the email? And why not the `username`? Why not supporting both?

Reply:
Simply because:
- it's what auth package requires from us
- you receive a letter on your email for password recovery. So it's not required to remember a username but if you do remember a username how would you know which email will receive the letter.
- it's like a standard, email is used everywhere for that.